packages/cli/src/command-implementations/lint-command.ts

```diff
@@ -14,6 +14,10 @@ export type LintCommandDefinition = typeof commandDefinitions.lint
 export class LintCommand extends ProjectCommand<LintCommandDefinition> {
   public async run(): Promise<void> {
     const projectDef = await this.readProjectDefinitionFromFS()
+    if (projectDef.bpLintDisabled) {
+      this.logger.warn('Linting is disabled for this project')
```

Comment:
**suggestion**: it might be worthwhile to explain _why_ linting is disabled; something like `Linting is disabled for this project because of a bplint directive. To enable linting, remove the "bplint-disable" directive from the project definition file`